### PR TITLE
Remove Cavy Tester - it has a new home!

### PR DESCRIPTION
**Pull request by AbigailMcP**
We're moving CavyTester to a separate repo due to issues with RN 0.63 and symlinking in the same repo.

Check it out here: https://github.com/pixielabs/cavy-tester

TODO:
- [x] Update README instructions re contributing to Cavy
- [x] Get tests running in CI

### CODE REVIEW

File: .watchmanconfig [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (jalada) [inline .watchmanconfig:1]: Presumably you can get shot of this now?

Comment (AbigailMcP) [reply .watchmanconfig:1]: I'm using wml to develop locally - and that's what I'm recommending in the cavy-tester readme, so it's nice not to copy these files.

Comment (jalada) [reply .watchmanconfig:1]: Oh wait sorry, was getting confused about watchman vs. wml. Ignore me!

**Comment by jalada** [comment]
We're holding this PR for a spell whilst we investigate some issues with newer versions of React Native & their abundance of newly native components causing a lot of Cavy spec helpers to stop working without `wrap()`.

**Pull request merged**